app/models/ability.rb

```diff
@@ -18,6 +19,9 @@ def initialize(current_user)
 
   def assign_basic_permission
     can :manage, Ckey2bibframe
+    can :read, UniUpdatesErrors
+    can :read, UniUpdMhldError
+    can :read, BatchRecordUpdate
```

Comment:
This now allows me to view the Batch Record Updates menu on the main page and the show actions on the batch record updates menu page when I remove my user from authorized_user. Thanks!

Reply:
Great!